monkey/infection_monkey/exploit/web_rce.py

```diff
@@ -83,13 +83,13 @@ def _exploit_host(self):
         if not ports:
             return False
         # Get urls to try to exploit
```

Comment:
Might as well remove this comment

Reply:
I'll leave it for the sake of consistency (It's not more trivial than the ones before it, and I think it helps dividing the flow into parts). If it really bothers you I'll erase it :D